src/fsharp/FSharp.DependencyManager/FSharp.DependencyManager.fsproj

```diff
@@ -19,16 +18,6 @@
     <PropertyGroup>
       <BuildOutputGroupLocation>$(BaseOutputPath)\$(Configuration)\$(TargetFramework)</BuildOutputGroupLocation>
     </PropertyGroup>
-
-    <ItemGroup>
```

Comment:
Double check that these assemblies end up in the Microsoft.FSharp.Compiler package, otherwise we'll likely have a problem at runtime.

Reply:
^^^ This still applies.